setup.py

```diff
@@ -1,24 +1,33 @@
 """python-vtzero setup."""
 
-import os
-
 from setuptools import setup
 from setuptools.extension import Extension
 from Cython.Build import cythonize
 
-os.environ["CC"] = 'clang++'
+with open("vtzero/__init__.py") as f:
```

Comment:
I'd rather write the version number here, and then expose it from the `__init__.py` using pkg_resources: https://setuptools.readthedocs.io/en/latest/pkg_resources.html

Reply:
I took a chance 😄